### ISSUE TITLE
Support camt.053

### ISSUE THREAD

**Issue by andrewshadura** [raphaelm/python-sepaxml#58]
Bank statements in camt.053 share some parts of the structure, so probably should be relatively easy to add support.

For your reference, a simple parser implementation can be found in https://github.com/tkarabela/okane

**Comment by raphaelm** [comment]
I don't think this library can be easily transformed from "generating" to "parsing" without a more or less full rewrite

**Comment by andrewshadura** [comment]
Right. I didn't realise it was only generating XML files.
In fact, for my usecase just generating would be enough, as I need to convert a CSV statement into a SEPA XML.

**Comment by raphaelm** [comment]
I don't personally plan on adding that, since I have no use for it, but I'd consider a well-tested PR for it
